src/types.ts

```diff
@@ -170,13 +170,14 @@ export interface FileGetPayload {
 export type QueryParams = string[] | { [key: string]: any }
 
 export interface TriggerPayload {
-  trigger_id: number
+  trigger_id: number | null
   trigger: Trigger
   table: string
   record_id: number
   application_id?: number
   event_code?: string
   data?: { [key: string]: any }
+  previewData?: Partial<ActionApplicationData>
```

Comment:
I would call it applicationDataOverride, I think it's a bit more clear

Reply:
Yup sure: 51dcdd72

This will definitely break the snapshot used for the front-end though, so you'll need to rename the parameter in there if I don't get to it first.

Reply:
Didn't break the snapshot, just had to change the parameter name in front-end. Good to go now.